parlai/core/torch_agent.py

```diff
@@ -44,51 +44,41 @@ class Batch(AttrDict):
     with additional fields if they would like, though we recommend calling the
     parent function to set up these fields as a base.
 
-    .. py:attribute:: text_vec
-
+    :param text_vec:
```

Comment:
any reason we still use `:attribute:` below in torch generator but not here?

Reply:
I will want to update that, good catch